evap/results/templates/results_course_detail.html

```diff
@@ -179,14 +178,27 @@ <h3>{{ course.name }} ({{ course.semester.name }})</h3>
                 {% endfor %}
             </div>
         </div>
-    {% endif %}
-
-    {% if course_sections_bottom %}
+        {% if course_sections_bottom %}
+            <div class="card card-outline-primary">
+                <div class="card-header">
+                    {% trans 'Course' %}
+                </div>
+                <div class="card-body">
+                    {% for section in course_sections_bottom %}
+                        {% include 'results_course_detail_questionnaires.html' %}
+                    {% endfor %}
+                </div>
+            </div>
+        {% endif %}
+    {% else %}
         <div class="card card-outline-primary">
             <div class="card-header">
                 {% trans 'Course' %}
             </div>
             <div class="card-body">
+                {% for section in course_sections_top %}
+                    {% include 'results_course_detail_questionnaires.html' %}
+                {% endfor %}
```

Comment:
I would have merged course_secions_top and course_sections_bottom in the view if there are no contributor_sections. Have you thought about that? I guess it would result in less code duplication, but it's possibly harder to understand.

Reply:
i'm not sure about that. in the end it's just a display thing and semantically having two lists is what we do, but both solutions are fine to me. can i have one more opinion from @karyon, please?

Reply:
right now we are splitting into the three lists in the view, which is a display thing already. splitting into three lists in the view and (if necessary) merging into one list in the template does not make *that* much sense, and personally i find the current solution to have too many branches/code paths in the template, also doing this in the view is probably much shorter, so i would support @He3lixxx's suggestion.